gsad/src/gsad_gmp.c

```diff
@@ -1255,11 +1256,27 @@ get_many (gvm_connection_t *connection, const char *type,
             }
           else if (str_equal (type, "result"))
             {
+              gchar *report_id_filter;
+              and_report_id = params_value (params, "_report_id");
```

Comment:
Shouldn't this get the `_and_report_id` parameter instead of `_report_id`?
```suggestion
              and_report_id = params_value (params, "_and_report_id");
```

Reply:
Of course it should be `_and_report_id`. Fixed.